src/crypto/hsm/nxp/CHIPCryptoPALHsm_SE05X_Spake2p.cpp

```diff
@@ -500,7 +496,8 @@ CHIP_ERROR Spake2pHSM_P256_SHA256_HKDF_HMAC::ComputeRoundTwo(const uint8_t * in,
 
 CHIP_ERROR Spake2pHSM_P256_SHA256_HKDF_HMAC::KeyConfirm(const uint8_t * in, size_t in_len)
 {
-    VerifyOrExit(state == CHIP_SPAKE2P_STATE::R2, error = CHIP_ERROR_INTERNAL);
+    CHIP_ERROR error = CHIP_ERROR_INTERNAL;
```

Comment:
move declaration where it is used  rather than on top (need to put this on line 519). We want to decrease variable scope as much as possible. Previous version of the code was correct.

Reply:
Sure. Done as Suggested